woocommerce/woocommerce-gutenberg-products-block
# Include address fields in the AddressForm component

This PR moves away from using core data for address fields, into the AddressForm component. This ensures labels are correct (based on optional/required status), and that the data is available at all times.

Fixes #1879
Fixes #1888

Extensibility is hindered, however, this is something we can iterate on in the future.

### How to test the changes in this Pull Request:

1. Go to checkout
2. Try changing countries 
3. Confirm fields change, labels update etc
